doc/man3/SSL_CTX_set1_curves.pod

```diff
@@ -75,17 +75,76 @@ to downgrade connections to a weaker algorithm.
 =back
 
 SSL_CTX_set1_groups_list() sets the supported groups for B<ctx> to
-string B<list>. The string is a colon separated list of group names, for example
-"P-521:P-384:P-256:X25519:ffdhe2048". The groups are used as in
+string B<list>. As an extension to SSL_CTX_set1_groups(), the names of the
+groups rather than their NIDs are used. The groups are used as in
 SSL_CTX_set1_groups(), described above. Currently supported groups for
 B<TLSv1.3> are B<P-256>, B<P-384>, B<P-521>, B<X25519>, B<X448>,
 B<brainpoolP256r1tls13>, B<brainpoolP384r1tls13>, B<brainpoolP512r1tls13>,
 B<ffdhe2048>, B<ffdhe3072>, B<ffdhe4096>, B<ffdhe6144> and B<ffdhe8192>. Support
-for other groups may be added by external providers, however note the discussion
-on TLS 1.3 selection criteria above. If a group name is preceded with the C<?>
-character, it will be ignored if an implementation is missing. The "DEFAULT"
-group name is used to select the default selection of groups. Group names that
-are preceded with the C<-> character will be removed from the selected groups.
+for other groups may be added by external providers.
+The group can be either the B<NIST> name (e.g. B<P-256>), some other commonly
+used name where applicable (e.g. B<X25519>, B<ffdhe2048>) or an OpenSSL OID name
+(e.g. B<prime256v1>). Group names are case sensitive.
```

Comment:
The surrounding text is changing in the `feature/ml-kem` branch of OpenSSL 3.5, which also introduces hybrid PQ groups, ...
It is perhaps most helpful, if you're willing and able to rebase this PR against the feature branch, rather than `master`.

```suggestion
Each group can be either the B<NIST> name (e.g. B<P-256>), some other commonly
used name where applicable (e.g. B<X25519>, B<ffdhe2048>) or an OpenSSL OID name
(e.g. B<prime256v1>). Group names are case sensitive.
```

Reply:
Do you think it's something that should be merged into this feature branch instead of master ? It is independent of this feature really. I would still like to get everything clean first on master, but if it's still thought that this could be a good idea then I can look at it to see.

Reply:
@vdukhovni I disagree. This PR is useful on its own and is not dependent on the feature branch. So it should be kept as a PR against the master branch.

Reply:
Yes, this can also be used/tested with oqs-provider, not only with the ml-kem feature branch.